style.css

```diff
@@ -0,0 +1,310 @@
+@font-face {
+  font-family: 'Lato';
+  src: url('./fonts/Lato-Regular.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Cocogoose';
+  src: url('./fonts/Cocogoose-Classic-ExtraBold-trial.ttf') format('truetype');
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@800&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', serif;
+}
+
+.white-link {
+  color: #fff;
+}
+
+.white-link:hover {
+  color: #f7423f;
+}
+
+.black-link {
+  color: rgba(0, 0, 0, 0.65);
+}
+
+.black-link:hover {
+  color: #f7423f;
+}
+
+.nav-main {
+  background: url('./imgs/navbg.jpeg');
+}
+
+.navbar {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.nav-dark {
+  background: #414246;
+  height: 32px;
+}
+
+.nav-collapse-bar {
+  background: #414246;
+  font-size: 12px;
+  height: 0;
+}
+
+.navbar-collapse {
+  position: absolute;
+  z-index: 999;
+  top: 100%;
+  right: 0;
+  left: 0;
+  background-color: rgba(250, 242, 242, 0.9);
+  color: #000;
+  font-weight: bold;
+  padding: 10px;
+}
+
+.nav-collapse ul {
+  height: 32px;
+}
+
+.nav-logo {
+  width: 100px;
+  height: 65px;
+}
+
+.nav-logo img {
+  width: 100px;
+}
+
+.mr-30 {
+  margin-right: 30px;
+}
+
+.masters-btn {
+  background-color: transparent;
+  color: #f7423f;
+  border: #f7423f solid 4.1px;
+  border-radius: 0;
+}
+
+.masters-btn:hover {
+  background-color: #f34e3e;
+  color: rgba(0, 0, 0, 0.65);
+  border: #f34e3e solid 4.1px;
+}
+
+.menu-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: rgba(0, 0, 0, 0.65);
+  font-size: 30px;
+}
+
+.headline {
+  background: url('./imgs/container-bg.jpeg');
+  background-size: cover;
+  background-repeat: repeat;
+}
+
+.headline-div {
+  margin-top: 0;
+  padding-top: 70px;
+  padding-bottom: 60px;
+  margin-bottom: 40px;
+}
+
+.headline-div h2 {
+  color: #f34e3e;
+  font-size: 23.6px;
+  backdrop-filter: blur(8px);
+}
+
+.headline-div h1 {
+  color: #ec5242;
+  font-family: 'Cocogoose', serif;
+  font-weight: 400;
+  font-size: 34.6px;
+  backdrop-filter: blur(8px);
+}
+
+.headline-div p {
+  color: rgba(0, 0, 0, 0.65);
+  background-color: #fff;
+  margin-top: 27px;
+  padding: 10px;
+  box-shadow: 0 0;
+  border: 5.4px solid white;
+}
+
+.headline-div h3 {
+  margin-top: 20px;
+  color: rgba(0, 0, 0, 0.65);
+  font-family: 'Poppins', sans-serif;
+  font-size: 23.6px;
+  letter-spacing: 0;
+  padding: 6px;
+  font-weight: 900;
+  text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.35);
+}
```

Comment:
- [ ] Good job making the text bolder as requested, however, you have changed the font family being used here. Now you are using `Poppins` instead of `Lato`. Kindly use `Lato` but still maintain the changes you've made.

Before
![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/81932478/8c4c89d9-359c-4ca7-b3f4-31fa235b7dd9)

After
![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/81932478/72a47b25-8dd7-48e3-9bac-64fd64bf2afc)

#### The changes you have introduced are  good, we just need to follow the design guidelines by using the given font-family so as to meet the requirements